docs/api/endpoints.md

```diff
@@ -581,6 +581,37 @@ Status: 200 OK
 }
 ```
 
+## Unpproving a post
+
+```
+POST /api/posts/:post_id/unapprove (requires authentication)
+```
+
+__NOTE__
+- Only an Admin or a Moderator can unapprove a post that is approved but not published.
+- Error out in case of normal user tries to unapprove a post.
+- Error out in case of trying to unapprove a post that has not been approved yet.
+- Error out in case of trying to unapprove a post that has already been published.
+
```

Comment:
Where is the request body?

If it doesn't accept anything, then please mention that. It should be explicit to anyone reading the docs and they shouldn't be left wondering.